benchmarks/README.md

```diff
@@ -37,3 +37,15 @@ bash setup_deb.sh  # works on Debian-based systems
 bash download_models.sh  # uncomment preferred models in the file, by default llama3 q8_0 will be downloaded
```

Comment:
I would just change naming convention to "llama.cpp (commit)" from "unoptimized" and "Ampere optimized llama.cpp (release)" from optimized to be more explicit.